commit 3b132ea12603f4c4d9ccd89639acc81c4c9ec163
Author: Jamie Strandboge <jamie@ubuntu.com>
Date:   Tue Jan 21 14:40:52 2020 -0600

    setup.py: look for iptables in /usr/sbin first
    
    Historically iptables was in /sbin, then later also symlinked from
    /usr/sbin/iptables to /sbin/iptables. Debian bullseye moves iptables to
    /usr/sbin with no symlink in /sbin except on upgrades. To accommodate
    buildds that may still have the old iptables, search /usr/sbin first
    
    References:
    https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=921680

Index: ufw-0.36/setup.py
===================================================================
--- ufw-0.36.orig/setup.py
+++ ufw-0.36/setup.py
@@ -249,7 +249,11 @@ iptables_exe = ''
 iptables_dir = ''
 
 for e in ['iptables']:
-    for dir in ['/sbin', '/bin', '/usr/sbin', '/usr/bin', '/usr/local/sbin', \
+    # Historically iptables was in /sbin, then later also symlinked from
+    # /usr/sbin/iptables to /sbin/iptables. Debian bullseye moves iptables
+    # to /usr/sbin with no symlink in /sbin except on upgrades. To accomodate
+    # buildds that may still have the old iptables, search /usr/sbin first
+    for dir in ['/usr/sbin', '/sbin', '/usr/bin', '/bin', '/usr/local/sbin', \
                 '/usr/local/bin']:
         if e == "iptables":
             if os.path.exists(os.path.join(dir, e)):
